client/src/component/Auth.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import TopBar from "./TopBar";
+
+const user = {
+	FName: 'Pierre',
+	LName: 'Smith',
+	Province: 'Ontario',
+	PostalCode: 'M4C 2R2',
+	City: 'Toronto',
+	role: 'Chef',
+}
+	
+export default class Auth extends Component {
```

Comment:
we will be using functional components with hooks as per [official React recommendations](https://reactjs.org/docs/hooks-faq.html#should-i-use-hooks-classes-or-a-mix-of-both).